server/users.js

```diff
@@ -41,6 +41,10 @@ const DEFAULT_TRAINER_SPRITES = [1, 2, 101, 102, 169, 170, 265, 266];
 /** @type {typeof import('../lib/fs').FS} */
 const FS = require(/** @type {any} */('../.lib-dist/fs')).FS;
 
+const MINUTES = 60 * 1000;
```

Comment:
Thinking out loud: we should probably revisit these constants once we have some data (ie. see how many users/staff would be marked away today and tweak these values to ensure they line up with our expectations?)

Reply:
yeah this is arbitrary for now and we can change it.